docs/doc/14-sql-commands/20-query-syntax/01-query-select.md

```diff
@@ -68,9 +70,46 @@ SELECT number AS `Total` FROM numbers(3);
 +--------+
 ```
 
-## EXCLUDE Parameter
+If you alias a column in the SELECT clause, you can use the alias in the WHERE, GROUP BY, and HAVING clauses:
+
+```sql
+SELECT number * 2 AS a FROM numbers(3) WHERE (a + 1) % 3 = 0;
```

Comment:
alias could also be used in select clause,  the latter expression could use the alias declared before.

```
select number * 2 as a, a * 2 as b from numbers(3)
```